### ISSUE TITLE
Deployment cache busting

### ISSUE THREAD

**Issue by grant-h** [CTFd/CTFd#912]
When making changes to deployed CSS and JS assets (in dev or prod), a hard refresh is required to see these. This appears to be due to Flask default caching.

```
> GET /themes/core/static/css/base.css HTTP/1.1
> Host: localhost:4000
> User-Agent: curl/7.47.0
> Accept: */*
>
< HTTP/1.1 200 OK
< Content-Length: 2773
< Content-Type: text/css; charset=utf-8
< Last-Modified: Thu, 07 Mar 2019 19:20:55 GMT
< Cache-Control: public, max-age=43200
< Expires: Thu, 21 Mar 2019 05:43:46 GMT
< ETag: "1551986455.78-2773-2390758808"
< Set-Cookie: session=9b4f1b3c-cbc0-456c-8660-f7e980870d17; HttpOnly; Path=/; SameSite=Lax
< Date: Wed, 20 Mar 2019 17:43:46 GMT
```

I figure most people also place CTFd behind a reverse proxy for HTTPS and load balancing, which could handle this caching. I wonder if it would be possible to either disable this caching by default and leave it up to the user?

**Comment by ColdHeat** [comment]
I believe the value you're looking for in config.py is `SEND_FILE_MAX_AGE_DEFAULT`?
http://flask.pocoo.org/docs/1.0/config/#SEND_FILE_MAX_AGE_DEFAULT

While I think this is the right setting I think this is something we should resolve generically so I'm going to leave this open to think about it.

**Comment by ColdHeat** [comment]
I looked at Ghost which resolves this by creating a string based on the server start time. 
https://ghost.org/docs/api/v2/handlebars-themes/helpers/asset/#description

**Comment by ColdHeat** [comment]
Closed in #1153
